packages/build/README.md

```diff
@@ -132,6 +132,36 @@ npm run build
 
     `converage:ci` sets up integration with [Coveralls](https://coveralls.io/).
 
+## A note on console logs printed by tests
+
+We consider (console) logging from tests as a bad practice, because such logs
+usually clutter the test output and make it difficult to distinguish legitimate
+error messages from the noise.
+
+By default, `lb-mocha` detects when the tests and/or the application tested is
```

Comment:
`is` -> `has`

Reply:
Good catch!

I am going to use `have`, because there are multiple subjects involved (`the tests and/or the application`).